docs/sources/structure/topic-types/landing-page/index.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2024-12-18
+description: Learn how to create a landing page.
+keywords:
+  - topic types
+  - template
+  - landing page
+menuTitle: Landing page
+review_date: 2024-12-18
+title: Landing page
+---
+
+<!-- vale Grafana.Gerunds = NO -->
+
+<!-- vale GGrafana.Gerunds = YES -->
+
+
+
+
+## Create landing pages
+
+Create a landing page as a starting point for customers to get access to the documentation they are looking for.
+
+To create a landing page, complete the following steps.
+
+1. Go to the folder and \_index.md file for your landing page topic.
+1. Double-check existing front matter. You can leave existing front matter as is.
+
+   If there is useful text in the existing landing page - add it/create an Introduction topic nested underneath the landing page.
+
+1. Add the `hero` shortcode to the front matter.
+
+   This shortcode becomes the banner at the top of the landing page.
+
+   Example:
+
+   ```markdown
+   hero:
+   title: Grafana Alerting
+   level: 1
+   image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
+   width: 100
+   height: 100
+   description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
+   ```
+
+   {{< admonition type="note" >}}
+   You may have to adjust the width or height to 100/110 depending on the spacing.
+   {{< /admonition >}}
+
+1. Add the `card-grid` shortcode to the front matter. This shortcode is for the tiles that appear below ## Explore.
+
+1. Add the `card-grid` shortcode to the front matter. This shortcode is for the tiles that appear below ## Explore.
```

Comment:
Line 51 and 53 are duplicates.

Question, is `card` deprecated?  If so, it's worth mentioning that here.

Reply:
thanks julie, fixed!